app/lib/main.dart

```diff
@@ -39,6 +40,13 @@ class Effektio extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
+      //To avoid redundancy, the whole textTheme of app is setup as Montserrat,
+      //so we can avoid using it explicitly in every Text Widget
```

Comment:
This would be a useful comment _on the PR description_ as it is an unrelated but useful change and would have helped me understand all the changes in the other files, that didn't seem to belong here.